jobs/openvpn/templates/etc/openvpn.conf.erb

```diff
@@ -26,6 +26,14 @@ server <%= p('openvpn.server') %>
 <% p('openvpn.push_routes').each do | route | %>
 push "route <%= route %>"
 <% end %>
+<% p('openvpn.push_dns').each do |dns| %>
+# ensure we can route to the DNS server over the connection
+push "route <%= dns %> 255.255.255.255"
+push "dhcp-option DNS <%= dns %>"
+<% end %>
+<% if_p('openvpn.push_dns_search_domains') do |domains| %>
+push "dhcp-option DOMAIN-SEARCH <%= domains.join(" ") %>"
```

Comment:
I think the [official/correct](https://community.openvpn.net/openvpn/wiki/Openvpn24ManPage) option name is `DOMAIN` and it only accepts a single domain. Do you have cross-platform experience with both of these changes?